registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -3392,6 +3392,38 @@
           "tokenManagerType": "mintBurn"
         }
       ]
+    },
+    "0x4373f265c4d680dec5635152187b505f3820570ad4d8bbf811ff93ae826323ed": {
+      "tokenId": "0x4373f265c4d680dec5635152187b505f3820570ad4d8bbf811ff93ae826323ed",
+      "deployer": "0xcE6084118756Da652F793fd52b45D6FCd48b52D8",
+      "originalMinter": null,
+      "prettySymbol": "CATBOY",
+      "decimals": 18,
+      "originAxelarChainId": "ethereum",
+      "tokenType": "canonical",
```

Comment:
is this "canonical" or "customInterchain"?

If you deployed token managers programmatically yourself to link the token as an interchain token, then it is "customInterchain". Only if you deploy it using the portal as an existing token, that's when it is "canonical." Please update.

Reply:
Corrected